tox.ini

```diff
@@ -61,7 +61,7 @@ deps:
 commands =
     pip install pip --upgrade
     pip install tox --upgrade
-    pyutils-jira --config-file-path jira.cfg --target-versions "vfuture,4.19.0,4.19.1,4.19.z,4.20.0" --verbose
+    pyutils-jira --config-file-path jira.cfg --target-versions "4.19.1,4.19.z" --verbose
```

Comment:
Should we add more z versions? Some bugs are intentionally targeted for later because they are not planned to be fixed in 4.19.1 for example
`4.19.1,4.19.2,4.19.3,4.19.4,4.19.5,4.19.z`